src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -282,13 +285,34 @@ static bool hasImplementation(AccessorDeclarationSyntax accessor)
                 return body != null;
             }
 
-            static bool containsFieldKeyword(SyntaxNode syntax)
+            static bool containsFieldExpressionInAccessor(AccessorDeclarationSyntax syntax)
+            {
+                var accessorDeclaration = (Syntax.InternalSyntax.AccessorDeclarationSyntax)syntax.Green;
+                foreach (var attributeList in accessorDeclaration.AttributeLists)
+                {
+                    var attributes = attributeList.Attributes;
+                    for (int i = 0; i < attributes.Count; i++)
+                    {
+                        if (containsFieldExpressionInGreenNode(attributes[i]))
```

Comment:
so is the idea that this way of visiting the syntax is skipping the attribute target?

Reply:
Yes, we're skipping the attribute target. I've since changed `containsFieldExpressionInGreenNode()` to look for `FieldExpression` rather than `FieldToken`, so this change is no longer strictly necessary, but it seemed reasonable to 
me to keep both changes.